projects/storefrontlib/cms-components/myaccount/my-coupons/my-coupons.component.ts

```diff
@@ -93,21 +93,20 @@ export class MyCouponsComponent implements OnInit, OnDestroy {
     );
     this.sortLabels = this.myCouponsComponentService.getSortLabels();
 
-    this.subscriptions
-      .add(
-        this.couponService
-          .getSubscribeCustomerCouponResultError()
-          .subscribe((error) => {
-            this.subscriptionFail(error);
-          })
-      )
-      .add(
-        this.couponService
-          .getUnsubscribeCustomerCouponResultError()
-          .subscribe((error) => {
-            this.subscriptionFail(error);
-          })
-      );
+    this.subscriptions.add(
+      this.couponService
+        .getSubscribeCustomerCouponResultError()
+        .subscribe((error) => {
+          this.subscriptionFail(error);
+        })
+    );
+    this.subscriptions.add(
+      this.couponService
+        .getUnsubscribeCustomerCouponResultError()
+        .subscribe((error) => {
+          this.subscriptionFail(error);
+        })
+    );
```

Comment:
Should the observer object be used here?

Reply:
No, only when using multiple callback functions is the listener object required.  When using a singular "next" callback function, we are not required to use a listener object.